packages/alert/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@littlespoon/alert",
+  "version": "1.0.0",
+  "description": "Little Spoon alert",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
+    "clean": "rm -rf coverage exports lib",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "lint:tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@littlespoon/theme": "1.13.0"
```

Update to latest version:

```suggestion
    "@littlespoon/theme": "1.13.1"
```